db/data/20220516105534_fix_funding_methods_for20192020.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class FixFundingMethodsFor20192020 < ActiveRecord::Migration[6.1]
+  UNDERGRAD_BURSARIES = [
+    OpenStruct.new(
+      training_route: TRAINING_ROUTE_ENUMS[:provider_led_undergrad],
+      amount: 9_000,
+      allocation_subjects: [
+        AllocationSubjects::MATHEMATICS,
+        AllocationSubjects::PHYSICS,
+      ],
+    ),
+    OpenStruct.new(
+      training_route: TRAINING_ROUTE_ENUMS[:opt_in_undergrad],
+      amount: 9_000,
+      allocation_subjects: [
+        AllocationSubjects::MATHEMATICS,
+        AllocationSubjects::PHYSICS,
+        AllocationSubjects::COMPUTING,
+        AllocationSubjects::MODERN_LANGUAGES,
+      ],
+    ),
+  ].freeze
+
+  def up
+    academic_cycle = AcademicCycle.find_by(start_date: "2019-09-01")
```

Comment:
@stevehook has got a migration PR on the go to change these so we'll need to be careful with the order of merging if we use the start date. Maybe better to get them all and use the one with the 2019 start year 🤷

Reply:
Updated to use `for_year` method which doesn't depend on exact start date.